serverless.yml

```diff
@@ -34,6 +34,7 @@ plugins:
 
 provider:
   name: aws
+  role: arn:aws:iam::819013376994:role/tf-${self:custom.service}-${self:custom.stage}-lambda-execution
```

Comment:
Can we do this instead via an `Fn::ImportValue`?

Might also want a simple CloudFormation export a la https://github.com/FormidableLabs/aws-lambda-serverless-reference/blob/master/terraform/main.tf#L120-L161 in `terraform-aws-serverless` too...

Reply:
Serverless doesn't run any CloudFormation intrinsics on this field 😭 It treats them as string literals

Reply:
I'm confused: how is this existing code working https://github.com/FormidableLabs/aws-lambda-graphql-server/blob/master/serverless.yml#L60-L61 ? (And I think it does...)

Reply:
Yeah, it works by proxying this from `serverless.yml`:

```yml
provider:
  # ...
  role:
    Fn::ImportValue: "aws-${self:custom.service}-${self:custom.stage}-LambdaRoleArn"

```

straight _into_ the built stack function in `.serverless/cloudformation-template-update-stack.json`:

```js
    "ServerLambdaFunction": {
      "Type": "AWS::Lambda::Function",
      "Properties": {
        // ...
        "FunctionName": "sls-graphql-server-sandbox-server",
        "Handler": "src/server/index.handler",
        "MemorySize": 128,
        "Role": {
          "Fn::ImportValue": "aws-graphql-server-sandbox-LambdaRoleArn"
        },
        // ...
      },
      // ...
    },
```

Reply:
Oh yeah I see the problem–I was trying to interpolate intrinsics _inside_ the string 🤦‍♂ will update